### PR TITLE
Reinstate support for PEP_REPO_PATH setting.

### PR DESCRIPTION
**Pull request by davidfstr**
This setting is referenced by PEP generation documentation
in both this repository (at pythondotorg/docs/source/pep_generation.rst)
and in the PEPs repository (at peps/README.rst).

### CODE REVIEW

**Comment by davidfstr** [comment]
Additional context: This setting is very useful for doing local drafting of PEPs, because it allows you to run the python.org website locally with any draft PEPs that you might be working on.

File: peps/management/commands/generate_pep_pages.py
Comment (merwok) [inline peps/management/commands/generate_pep_pages.py:55]: This if block doesn’t need to be in the exit stack context.

Comment (davidfstr) [reply peps/management/commands/generate_pep_pages.py:55]: The else clause of the if statement uses `stack` so I can't just move the entire if-statement up and out of the stack.

What kind of change were you thinking of exactly?

Comment (merwok) [reply peps/management/commands/generate_pep_pages.py:55]: Have the with block inside the else block

Comment (davidfstr) [reply peps/management/commands/generate_pep_pages.py:55]: Unfortunately that doesn't work. You'll notice that the exit stack is used to clean up files decompressed to `artifacts_path` which is used by the paragraphs following the if-statement. If I limit the scope of the exit stack to the else-clause it deletes the decompressed files before the rest of the command can read them.

And indeed, making that change anyway caused `python3 manage.py test peps` to start failing.

So I don't recommend moving the ExitStack around. Is there a particular concern of yours for having a single extra variable assignment inside the ExitStack that was there before?

Comment (merwok) [reply peps/management/commands/generate_pep_pages.py:55]: Ah I hadn’t noticed the uses of `stack` so far down!

**Comment by davidfstr** [comment]
Any other items blocking merge?

**Comment by davidfstr** [comment]
@merwok Did you still want to review this or should I seek someone else with more bandwidth?

**Comment by merwok** [comment]
I think one of the pydotorg maintainers should review this.

**Comment by davidfstr** [comment]
How do I determine who the pydotorg maintainers are? Is there a recommended way of getting their attention?

**Comment by merwok** [comment]
Checking who has merged PRs I guess.  I’ll tag ewdurbin on Monday!

**Comment by davidfstr** [comment]
Hello recent mergers @ewdurbin and @berinhard , this PR has been awaiting review since late Feb. Any chance one of you could take a look at it?

**Review by ewdurbin — APPROVED** [review]

**Comment by davidfstr** [comment]
Thanks @ewdurbin !

**Pull request merged**
